ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/serial-number.md

```diff
@@ -0,0 +1,18 @@
+<!--
+  ~ Copyright (c) 2025 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+=== "Table"
+
+    | Variable | Type | Required | Default | Value Restrictions | Description |
```

Comment:
should it be in its own table? I don't see this table included in the main doc so I feel like we would lose this

Reply:
addressed in one of Claus comments